apps/twa/src/libs/appSdk.ts

```diff
@@ -55,6 +55,9 @@ export class TwaAppSdk extends BaseApp {
     };
 
     openPage = async (url: string) => {
+        if (!url.startsWith('http')) {
+            throw new Error('Invalid url');
+        }
         if (url.includes('t.me')) {
             this.utils.openTelegramLink(url);
```

Comment:
could you please change native telegram utils to custom function:

function safeWindowOpen(url: string) {
  window.open(url, '_blank', 'noreferrer');
}